readme.md

```diff
@@ -386,6 +386,7 @@
 
 ## Entertainment
 
+- [Flash Briefings](https://github.com/davidsouza/awesome-flash-briefings) - Alexa Flash Briefings
```

Comment:
Should include a `#readme` at the end of the URL, so that it points users to the README file directly (see others in the list).

Reply:
Added